torch_geometric/datasets/generators/motif.py

```diff
@@ -0,0 +1,19 @@
+import torch
+
+
+class Motif:
+    def __init__(self, structure: str):
+        self.edge_index = None
```

Comment:
perhaps we could document what kind of "named" motifs are available so far (house). 

When people add more named motifs, they could modify in the documentation.

Also mention a todo that in future, we could allow edge_index or networkx objects for custom motifs other than named motifs.